Fixed keyswitch enum variables spelling mistakes

itu-itis24-abdulloev23 opened the pull request:
The keyswitching_type enum had wrong spelling for its variables,
keyswitching was spelled as keyswithing.

Alisah-Ozcan commented:
Thanks for the PR! Your branch isn’t up-to-date with main. Please update it and check CONTRIBUTING.md for guidelines. Let me know if you need help!

itu-itis24-abdulloev23 commented:
> Thanks for the PR! Your branch isn’t up-to-date with main. Please update it and check CONTRIBUTING.md for guidelines. Let me know if you need help!

I have updated the branch as you asked. Additionally I fixed the spelling mistake in the readme.md file that I missed last time
